<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeEditor_form</class>
 <widget class="QWidget" name="CodeEditor_form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0">
       <item>
        <widget class="QPushButton" name="pushButton_code_compileApp">
         <property name="text">
          <string>编译App</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_code_compileDll">
         <property name="text">
          <string>编译动态库</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_code_run">
         <property name="text">
          <string>运行App</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_code_save">
         <property name="text">
          <string>保存文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_code_open">
         <property name="text">
          <string>打开文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_code_test">
         <property name="text">
          <string>测试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_load_template">
         <property name="text">
          <string>加载模板</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_saveDll">
         <property name="text">
          <string>动态库保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_editor" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_messageClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_message"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_parameter"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
